<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rarfile_form</class>
 <widget class="QWidget" name="rarfile_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>408</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>文件解压</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="open_dir_btn">
     <property name="text">
      <string>打开目录</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="path_le"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListWidget" name="file_lw"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QRadioButton" name="other_rbtn">
        <property name="text">
         <string>解压到指定目录</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="other_path_le">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="cur_rbtn">
        <property name="text">
         <string>解压当前目录</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="self_rbtn">
        <property name="text">
         <string>解压到单独目录</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwd_le"/>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>136</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="unrar_btn">
     <property name="text">
      <string>解压</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
